smartpower3/screen.cpp

```diff
@@ -189,14 +189,14 @@ void Screen::checkOnOff()
 			channel[0]->off();
 			channel[1]->off();
 			tft.fillRect(0, 0, 480, 320, TFT_BLACK);
-			setting->setBacklightLevel(0);
+			setting->turnOffBacklight();
```

Comment:
This is the important change (and the so-called method).

I guess the author supposed `setBacklightLevel(0)` sets backlight level to `0` which was not the case. In this case, the zero was index number of array of backlight value presets (bl_value/bl_value_preset).

It just goes to show that naming items in a correct and unambiguous way is important - hence the renaming.